Fails to compile: 'title' is defined as an accessor... TabPaneComponent

## Version
3.0.1

## Environment
Angular 10.1 Zorro mobile 3.0.1

## Reproduction link
[https://stackblitz.com/edit/angular-ivy-zsymf2?file=package.json](https://stackblitz.com/edit/angular-ivy-zsymf2?file=package.json)

## Steps to reproduce
$ ng new PROJECT_NAME
$ cd PROJECT_NAME
$ ng add ng-zorro-antd-mobile --i18n=en_US
$ ng serve

## What is expected?
The app should compile and serve

## What is actually happening?
App fails to compile:

ERROR in node_modules/ng-zorro-antd-mobile/tab-bar/tab-bar-item.component.d.ts:10:5 - error TS2610: 'title' is defined as an accessor in class 'TabPaneComponent', but is overridden here in 'TabBarItemComponent' as an instance property.

10     title: string;
       ~~~~~
## Other?
If I go to node_modules/ng-zorro-antd-mobile/tab-bar/tab-bar-item.component.d.ts and comment out the line 12 (title: string) the app compiles

<!-- generated by ng-zorro-mobile-issue-helper. DO NOT REMOVE -->

--- a/components/tab-bar/tab-bar-item.component.ts
+++ b/components/tab-bar/tab-bar-item.component.ts
@@ -17,8 +17,6 @@ export class TabBarItemComponent extends TabPaneComponent {
   @Input()
   key: string = '';
   @Input()
-  title: string = '';
-  @Input()
   dot: boolean = false;
   @Input()
   badge: number | string = null;

